CITATION.rst

```diff
@@ -3,20 +3,21 @@ Acknowledging or Citing SunPy
 
 If you have used SunPy in your scientific work we would appreciate it if you would acknowledge it.
 The continued growth and development of SunPy is dependent on the community being aware of the use SunPy.
-If you use SunPy, we therefore ask that you acknowledge SunPy appropriately in a publication, presentation, poster, or talk.
 
--  **For a publication**, we recommend the following line be added to
-   the conclusion or acknowledgements:
+Publications
+------------
+
+Please add the following line to the conclusion or acknowledgements:
 
    *This research has made use of SunPy, an open-source and free
    community-developed solar data analysis package written in Python
    (citation).*
 
-   The citation is to the `SunPy v0.5 paper`_/`arXiv(open access)`_. If
-   the journal allows please also include a link to sunpy.org. If you
-   have the time, please email us to let us know about your paper, as we
-   maintain a `public list`_ of papers on `Zotero`_. A BibTeX entry for
-   LaTeX users is:
+The citation should be to the `SunPy v0.5 paper`_/`arXiv(open access)`_.
```

Comment:
Interesting... we are not asking to say which version of sunpy they are using? and tell them to add the zenodo reference?

Reply:
I thought we already had this in this file, we should definitely add it.